testing/environments/minimal/docker-compose.yml

```diff
@@ -91,6 +100,9 @@ services:
       KAFKA_REST_API_ENDPOINTS_BLOCKLIST: "api.v3.acls.*"
       KAFKA_REST_BOOTSTRAP_SERVERS: "kafka-1:9191"
       KAFKA_REST_LISTENERS: "http://0.0.0.0:9391"
+      KAFKAREST_HEAP_OPTS: "-Xms1g -Xmx1g -XX:MetaspaceSize=96m -XX:+UseG1GC \
```

Comment:
optional: if it's easy to comment or link to somewhere to better understand the rational for these settings, that'd be good. I'm assuming you're optimizing things.